README.md

```diff
@@ -22,3 +22,55 @@ Some of the tools and databases we're using were developed externally and cannot
 Tools requiring manual installation are listed and linked below:
 * [HH-suite 3.0](https://github.com/soedinglab/hh-suite)
 * [metaPSICOV](http://bioinfadmin.cs.ucl.ac.uk/downloads/MetaPSICOV/)
+
+## Naming conventions
+
+### Filenames
+All filenames are in the form: `GenomeID`\_`GeneID`\_`ResiduesFrom`-`ResiduesTo`.  
+For example, `CP003179.1_3319` means gene `3319` from genome `CP003179.1` (Sulfobacillus acidophilus DSM 10332), or `CP003179.1_3319_1-60` means amino acids 1 to 60 from that gene.
+
```

Comment:
it might be worth to give the reader the hint that those files are going to contain protein sequences and not DNA sequences of the gene as a subset from the genome.